src/components/navigation/navigation-item/NavigationItem.tsx

```diff
@@ -160,6 +160,7 @@ export class NavigationItem extends Component<Props, State> {
     } = data;
     return (
       <div
+        tabIndex={1}
```

Comment:
I hate using tabIndex different than -1 and 0. Is this necessary? Can't we use 0?